build.gradle

```
@@ -21,8 +27,11 @@ repositories {
 dependencies {
 	implementation 'org.springframework.boot:spring-boot-starter-data-jpa'
 	implementation 'org.springframework.boot:spring-boot-starter-web'
+	implementation 'org.springframework.boot:spring-boot-starter-validation'
 	implementation 'mysql:mysql-connector-java'
 	implementation 'junit:junit:4.13.1'
+	implementation "com.querydsl:querydsl-jpa:${queryDslVersion}"
+	implementation "com.querydsl:querydsl-apt:${queryDslVersion}"
```

implementation이나 compileOnly는 이렇게 하나로 묶어서 사용할 수도 있어요
```suggestion
	implementation (
	        'org.springframework.boot:spring-boot-starter-data-jpa',
        	'org.springframework.boot:spring-boot-starter-web',
        	'org.springframework.boot:spring-boot-starter-validation',
        	'mysql:mysql-connector-java',
        	'junit:junit:4.13.1',
        	'com.querydsl:querydsl-jpa:${queryDslVersion}',
                'com.querydsl:querydsl-apt:${queryDslVersion}'
	)
```

---

저는 개인적으로 한 괄호에 묶는거 보단 따로 있는게 가독성 좋은거 같아요! 
아니면 비슷한것끼리 묶거나

---

허허 다양한 피드백이네 수정안해야징